### ISSUE TITLE
Allow empty array for `verifiableCredential` in a VP?

### ISSUE THREAD

**Issue by dlongley** [digitalbazaar/bedrock-validation#48]
We require one VC if `verifiableCredential` is present -- should we allow it to be an empty array instead?

**Comment by gannan08** [comment]
Yes, an empty array is valid. I expect those using strongly typed languages to always serialize an array of size [0,N].
